rabbitmq_amqp_python_client/message_ack.py

```diff
@@ -0,0 +1,50 @@
+from typing import Dict
+
+from .qpid.proton._data import PythonAMQPData
+from .qpid.proton._delivery import Delivery
+from .qpid.proton._events import Event
+
+
+class MessageAck:
```

Comment:
I would remove this, and add a parameter in our Handler class, like `context` so in this way it would be like Java and .NET:
```
  (context, message) =>
    {
        Trace.WriteLine(TraceLevel.Information, $"[Consumer] Message: {message.Body()} received");
        context.Requeue();
  ```